astitodo/main.go

```diff
@@ -34,7 +35,15 @@ func main() {
 
 		// Filter results for assignee
 		if *assignee != "" {
-			todos = todos.AssignedTo(*assignee)
+			assignees := strings.Split(*assignee, ",")
+
+			var filteredTODOs astitodo.TODOs
+
+			for _, v := range assignees {
```

Comment:
You should also update testdata and test `AssignedTo` can indeed retrieve TODOs of several assignees